### DISCUSSION TITLE
Loco??

### DISCUSSION THREAD

**Discussion by karlscholz** [bitcraze/discussions#182] [Q&A] [answered]
What does Loco mean? Low cost? Could'nt find anything anywhere where the term Loco, or its origin is explained. Is it even an acronym?

**Comment by krichardsson** [comment, marked as answer]
"Loco" means "crazy" in Spanish, and is similar to the start of "Localization". We thought it was a nice parallel to the "crazy" in Crazyflie.

**Reply by karlscholz** [reply]
Ah, makes sense. :D thanks!

**Reply by krichardsson** [reply]
No worries! 

If you're happy with the answer, please click the "Mark as answer" button to make it more visible to other users. Thanks!

**Reply by karlscholz** [reply]
of course.
By the way do you know where I can find a specific code documentation for the loco system?
I found these:
- https://www.bitcraze.io/documentation/system/positioning/loco-positioning-system/
- https://www.bitcraze.io/documentation/repository/lps-node-firmware/master/
- https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/

But none of them are very close to the code. Is there another documentation that helps me understand better which function calls what and how it is implemented in the main system loop or do you have any tips for me on how and where to start with reverse engineering the code?

**Reply by knmcguire** [reply]
Please make a new discussion if you already selected an answer in this thread. The title should be descriptive to the initial requrest

**Reply by karlscholz** [reply]
will do, sry I'm still a bit new to github :)
